backend/grpc/protobuf/data.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package datapb;
+
+message DataRequest {
+    string filename = 1;
+}
+
+message Segment {
+    string country = 1;
+    string device = 2;
+    repeated string dates = 3;
+    repeated int64 inventories = 4;
```

Comment:
Comment on what are the values here?

Reply:
According to the protobuf documentation 'The " = 1", " = 2" markers on each element identify the unique "tag" that field uses in the binary encoding.'

Reply:
Sorry, I wasn't specific enough. Please, comment in the proto file what int64 inventories store.

Reply:
Oh okay, I already did that in the update of the CL.